spec/virtual_includes_spec.rb

```diff
@@ -0,0 +1,81 @@
+describe ActiveRecord::VirtualAttributes::VirtualIncludes do
+  before do
+    Author.destroy_all
+    Book.destroy_all
+    Author.create_with_books(3).books.first.create_bookmarks(2)
+  end
+
+  context "virtual column" do
+    it "as Symbol" do
+      expect { Author.includes(:nick_or_name).load }.to match_query_limit_of(1)
+    end
+
+    it "as Array" do
+      expect { Author.includes([:nick_or_name]).load }.to match_query_limit_of(1)
+      expect { Author.includes([:nick_or_name, :bookmarks]).load }.to match_query_limit_of(3)
+    end
+
+    it "as Hash" do
+      expect { Author.includes(:nick_or_name => {}).load }.to match_query_limit_of(1)
+      expect { Author.includes(:nick_or_name => {}, :bookmarks => :book).load }.to match_query_limit_of(4)
+    end
+  end
+
+  context "virtual reflection" do
+    it "as Symbol" do
+      expect { Author.includes(:named_books).load }.to match_query_limit_of(2)
+    end
+
+    it "as Array" do
+      expect { Author.includes([:named_books]).load }.to match_query_limit_of(2)
+      expect { Author.includes([:named_books, :bookmarks]).load }.to match_query_limit_of(3)
+    end
+
+    it "as Hash" do
+      expect { Author.includes(:named_books => {}).load }.to match_query_limit_of(2)
+      expect { Author.includes(:named_books => {}, :bookmarks => :book).load }.to match_query_limit_of(4)
+    end
+  end
+
+  # FIX
```

Comment:
FYI:  There any context with this "FIX" comment?  It just disappears 2 commits later.

What did you want to fix?  Was it fixed?

Reply:
thnx. I added context around it.

it had an miq column in there that wasn't valid